src/devices/oregon_scientific.c

```diff
@@ -868,7 +922,7 @@ static char *output_fields[] = {
 r_device oregon_scientific = {
         .name        = "Oregon Scientific Weather Sensor",
         .modulation  = OOK_PULSE_MANCHESTER_ZEROBIT,
-        .short_width = 440, // Nominal 1024Hz (488µs), but pulses are shorter than pauses
+        .short_width = 440, // Nominal 1024Hz (488Âµs), but pulses are shorter than pauses
```

Comment:
UTF8/Latin1 conversion problems introduced an invalid code here. Best to replace `µs` with `us`